apps/src/components/map-layers/map-legend-forecast-s2d.tsx

```diff
@@ -152,14 +152,14 @@ const ProbabilityStatement = (props: ProbabilityStatementProps) => {
 	);
 };
 
-export interface MapLegendInnerS2DProps {
+export interface MapLegendForecastS2DProps {
 	data: ColourQuantitiesMap;
 }
 
-export type MapLegendInnerS2D = typeof MapLegendInnerS2D;
+export type MapLegendForecastS2D = typeof MapLegendForecastS2D;
 
-export const MapLegendInnerS2D = (
-	props: MapLegendInnerS2DProps
+export const MapLegendForecastS2D = (
+	props: MapLegendForecastS2DProps
 ): JSX.Element => {
```

Comment:
See other comment about the `JSX` namespace being deprecated.


```suggestion
) => {
```